source/sites/publicodes/avantages.yaml

```diff
@@ -0,0 +1,9 @@
+- illustration: 🪟
```

Comment:
**question**: is this file in Yaml because it's used for Google metadata or something similar?

Reply:
no, the reason for it being in yaml is to make it accessible to anyone to edit :) 
more than JSON